cross/rtmpdump/digests

```diff
@@ -0,0 +1,3 @@
+rtmpdump-git550a0caea3cd3987ae3ce7eb20d0b4512dd064ac.tar.gz.wrong SHA1 b03b3a3702def3b6ea3336d081a2668159e6c5a7
```

Comment:
Remove the digests file when using `PKG_GIT_HASH`, there's no point in having a digests file: it'll be wrong every time.